requirements.txt

```diff
@@ -9,14 +9,15 @@ cookiecutter==2.4.0
 Django==4.2.10
 django-browser-reload==1.12.0
 django-tailwind==3.6.0
+python-dotenv
```

Comment:
redundant (seen on 22/23)

Reply:
I don't have any idea why but pip refused to let me download dotenv. It gave me an error until I did python-dotenv

Reply:
I had the same error!

Reply:
i mean the other one is also python-dotenv
let's delete this one (line 12, your addition) and make the other one (seen on 22/23) non-version dependent (remove '==1.0.1')

Reply:
Fixed